packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -330,16 +326,16 @@ class FuchsiaDevice extends Device {
 
     // Try Start with a fresh package repo in case one was left over from a
     // previous run.
-    final Directory packageRepo =
-        globals.fs.directory(globals.fs.path.join(getFuchsiaBuildDirectory(), '.pkg-repo'));
+    final Directory packageRepo = globals.fs.directory(
+        globals.fs.path.join(getFuchsiaBuildDirectory(), '.pkg-repo'));
```

Comment:
Please revert this style only nit per: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo